### DISCUSSION TITLE
Expose minio-operator tenant console via nginx ingress. The request signature we calculated does not match… you provided. Check your key and signing method.

### DISCUSSION THREAD

**Discussion by vaassi** [minio/minio#18505] [Q&A]
Hello, community!

I successfully deploy minio-operator via helm chart and create tenant, but can't correctly expose tenant console via nginx ingress. I can access console login page but when I'll try to enter login and password, I get `The request signature we calculated does not match… you provided. Check your key and signing method.`

If I use NodePort to access console, I can login.

I'll try different ingress annotation settings

```
nginx.ingress.kubernetes.io/ssl-passthrough: 'true'
nginx.ingress.kubernetes.io/upstream-vhost: minio
```
also enable "use-forwarded-headers": "true" global ingress setting, but nothing helps me to solve the issue.

I see some discussions about this issue that ingress not send correct headers to backend, but headers seems to be fine.
So how I can understand what is the source of my issue or may be get some debug info to understand the problem. Thanks.

**Comment by vaassi** [comment]
try to use latest version of minio `RELEASE.2023-11-20T22-40-07Z` and operator `v5.0.11` without any success

right now configuration is below:

ingress global:
![image](https://github.com/minio/minio/assets/5107627/9297bb8e-2935-46fb-8458-1729923ec5f0)

ingress local:
![image](https://github.com/minio/minio/assets/5107627/fcf6e365-765d-48fd-85b8-eea638322547)


I try to add `proxy_set_header Host $http_host;` but get 400 Bad request error.

**Reply by pawan-dhaliwal** [reply]
I am having the same issue, setting proxy_set_header Host $host results in error 400, and without results in the signature mismatch 403.  Was anyone able to get this working with NGINX Ingress on K8S?
